Engine/Source/ModuleScene.cpp

```diff
@@ -38,6 +39,30 @@ bool ModuleScene::Init()
 	return true;
 }
 
+GameObject* ModuleScene::FindGameObjectWithTag(std::string tagname)
+{
+	for (GameObject* child : mRoot->GetChildren())
+	{
+		if (child->GetTag()->GetName() == tagname) {
+			return child;
+		}
+	}
+
+	return nullptr;
+}
+
+std::vector<GameObject*> ModuleScene::FindGameObjectsWithTag(std::string tagname)
+{
+	std::vector<GameObject*> foundGameObjects = std::vector<GameObject*>();
+	for (GameObject* child : mRoot->GetChildren())
+	{
+		if (child->GetTag()->GetName() == tagname) {
+			foundGameObjects.push_back(child);
+		}
+	}
+	return foundGameObjects;
+}
+
```

Comment:
These methods should be recursive. That means that it will not find objects that are two or more levels below the parent.

Reply:
fixed